libraries/AP_Camera/AP_Camera_SoloGimbal.h

```diff
@@ -1,20 +1,24 @@
 #pragma once
 
-#include <AP_Mount/AP_Mount.h>
 #include <GCS_MAVLink/GCS_MAVLink.h>
 
 class AP_Camera_SoloGimbal {
 public:
 
-#define STATUS_NO_GOPRO             0
-#define STATUS_INCOMPATIBLE_GOPRO   1
-#define STATUS_GOPRO_CONNECTED      2
-#define STATUS_GOPRO_ERROR          3
-#define CAPTURE_MODE_VIDEO          0
-#define CAPTURE_MODE_PHOTO          1
-#define CAPTURE_MODE_BURST          2 //Burst only for Hero 3+
-#define CAPTURE_MODE_TIMELAPSE      3
-#define CAPTURE_MODE_MULTISHOT      4 //Multishot only for Hero4 
+  enum STATUS {
```

Comment:
`enum class`, @khancyr  :-)

Reply:
We cannot as mavlink message need uint8_t and we aren't permissive on conversion anymore